### ISSUE TITLE
Set window resolution

### ISSUE THREAD

**Issue by Tazovsky** [rstudio/webdriver#51]
Is it possible to set window resolution? Or if it's not, it would be very useful to be able to set it.

**Comment by bersbersbers** [comment]
Yes, compare https://github.com/rstudio/shinytest/blob/d0f6d737919b542c6a1e8bbd0ea05f9e4b3f5334/R/shiny-driver.R#L115-L123
